packages/nitro-protocol/test/contracts/TrivalApp/validTransition.test.ts

```diff
@@ -29,7 +29,7 @@ function getRandomVariablePart(): VariablePart {
 }
 
 beforeAll(async () => {
-  trivialApp = await setupContracts(provider, TrivialAppArtifact);
+  trivialApp = await setupContracts(provider, TrivialAppArtifact, process.env.TRIVIAL_APP_ADDRESS);
```

Comment:
If we're going to start using environment variables to share addresses for things in tests (which by the way seems unnecessary for unit tests as we discussed on standup), can we add a file which reads them from the environment and provides typed + validated accessors? For example, `environment.ts` might read `process.env. TRIVIAL_APP_ADDRESS` and if it sees `undefined` log an error in the console?